arviz/stats/stats.py

```diff
@@ -12,9 +12,8 @@
 from scipy.optimize import minimize
 import xarray as xr
 
+from ..plots.plot_utils import *
```

Comment:
Make the import explicit `from ..plots.plot_utils import _fast_kde, get_bins`

Reply:
@OriolAbril I already tried that, it is showing import error and the tests are not running

Reply:
`*` will surely not import `_fast_kde` because `*` only imports public functions. Using explicit import should work though, just like it worked with `from ..plots.kdeplot import _fast_kde` which is also on another folder). Could there have been a typo? I currently cannot test it nor work on this locally, this is as much as I can do for now.

Reply:
@OriolAbril I'll explicit import the function, so you can review!